srcpkgs/tuptime/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'tuptime'
+pkgname=tuptime
+version=5.2.4
+revision=1
+archs="*"
+build_style=fetch
+depends="python3"
+short_desc="Track and report system uptime and downtime"
+maintainer="Ricardo Fraile <r@rfmoz.eu>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/rfmoz/tuptime"
+changelog="https://raw.githubusercontent.com/rfmoz/tuptime/master/CHANGELOG"
+distfiles="https://github.com/rfmoz/tuptime/archive/refs/tags/${version}.tar.gz"
+checksum=d4038c5efb561ce4672f1b60025c6099fa243bed10063921ff9d1390cf5de863
+system_accounts="_tuptime"
+_tuptime_homedir="/var/lib/tuptime"
+_tuptime_descr="Tuptime execution user"
+make_dirs="/var/lib/tuptime   0755 _tuptime _tuptime"
+_pkgnv="${pkgname}-${version}"
+
+do_extract() {
```

Comment:
If you drop `build_style` and `do_extract()`, it'll just extract to `$wrksrc`. Then, in `do_install()`, you can just `vbin src/tuptime`, for example.

Reply:
Ok, done